.github/workflows/nightly.yml

```diff
@@ -1,9 +1,9 @@
 name: Nightly
 
-on:
-  schedule:
-    # every night at midnight
-    - cron:  '0 0 * * *'
+on: [pull_request]
+#   schedule:
+#     # every night at midnight
+#     - cron:  '0 0 * * *'
```

Comment:
Why have you removed the nightly cron? Is that temporary? Is that a draft PR?

Reply:
Yeah, just to make sure it works as expected, will revert in the upcoming commit(before merge) as soon as expected normal behavior is observed. The PR is complete, I was just testing the changes here instead of seeing the result of it when nightly runs in real, if we observe any error/problem in _simulated nightly build_ here we'll fix it right-away otherwise revert these commented lines.

Reply:
Then probably you can fix it now?

Reply:
https://github.com/kiwix/kiwix-android/pull/2586/commits/b59aaf1ff6fcbb26b4405be34ab6734c80ff944d